static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js

```diff
@@ -0,0 +1,77 @@
+define([
+    'lodash/utilities/template',
+    'common/modules/commercial/contributions-utilities',
+    'raw-loader!common/views/acquisitions-epic-testimonials.html',
+    'raw-loader!common/views/acquisitions-epic-equal-buttons.html',
+    'svgs/icon/quote.svg'
+], function (
+    template,
+    contributionsUtilities,
+    acquisitionsEpicTestimonialsTemplate,
+    acquisitionsEpicEqualButtonsTemplate,
+    quoteSvg
+) {
+
+    function greatBritainLocationCheck() {
```

Comment:
I don't think this is right - if you look at https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js#L106 you can see it just wants a function it can call, not a function that it has to call to get a function and then call that function

so I would've though you just want
```javascript
function greatBritainLocationCheck(userCountry) {
  return userCountry === 'GB';
}
```
also naming `isUserCountryGreatBritain` might be nicer, it's always good to have functions which return a boolean start with `is`